torch/testing/_internal/common_methods_invocations.py

```diff
@@ -6583,9 +6584,12 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                        # def the_method(i0):
                        #     return torch.polygamma(i0, 1)
                        #            ~~~~~~~~~~~~~~~ <--- HERE
-                       SkipInfo('TestCommon', 'test_variant_consistency_jit'),),
+                       SkipInfo('TestCommon', 'test_variant_consistency_jit'),
+                       SkipInfo('TestCommon', 'test_jit_alias_remapping'),
```

Comment:
Similar error as compared to `test_variant_consistency_jit`. This is because of the ordering of args in `polygamma` (`int, tensor` instead of tensor first).

```
======================================================================
ERROR: test_jit_alias_remapping_polygamma_polygamma_n_0_cpu_float32 (__main__.TestCommonCPU)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/krshrimali/git/krshrimali/pytorch/torch/testing/_internal/common_device_type.py", line 297, in instantiated_test
    raise rte
  File "/home/krshrimali/git/krshrimali/pytorch/torch/testing/_internal/common_device_type.py", line 292, in instantiated_test
    result = test_fn(self, *args)
  File "/home/krshrimali/git/krshrimali/pytorch/torch/testing/_internal/common_device_type.py", line 266, in test_wrapper
    return test(*args, **kwargs)
  File "/home/krshrimali/git/krshrimali/pytorch/test/test_ops.py", line 534, in test_jit_alias_remapping
    scripted = torch.jit.CompilationUnit(script)._fn
RuntimeError: 
Arguments for call are not valid.
The following variants are available:
  
  aten::special_polygamma(int n, Tensor self) -> (Tensor):
  Expected a value of type 'Tensor' for argument 'self' but instead found type 'int'.
  
  aten::special_polygamma.out(int n, Tensor self, *, Tensor(a!) out) -> (Tensor(a!)):
  Expected a value of type 'Tensor' for argument 'self' but instead found type 'int'.

The original call is:
  File "<string>", line 3

                        def _fn(t0, s0: int):
                            return variant(t0, s0)
                                   ~~~~~~~ <--- HERE
                    


----------------------------------------------------------------------
Ran 245 tests in 0.253s

FAILED (errors=1, skipped=223)
```

Reply:
I'm not sure if a comment is needed here since the reasoning looks similar for all the skips. But I think an issue should be raised for this one.

cc: @mruberry @kshitij12345

Reply:
This is a known failure. consistency_jit doesn't work when lambda is used. (I think there is already an issue tracking it since there are so many of these skips. Not 100% sure though)